### PR TITLE
Poker Hands Functions 7-10

### PR DESCRIPTION
**Pull request by Jedi-Rogue1**
I added a ".html" file this time to prove that my test works. I did the "High Card", "Pairs", "2 Pairs", and "3 of a Kind" Functions all under a new class I created called "PokerPlayer".

### CODE REVIEW

**Review by vondojr — APPROVED** [review]
I think all of this looks good! Good Organization and it seems like all three functions go together nicely! 

Maybe next time add a few more comments so it is easier to read through everything.

**Comment by mcmillanjy** [comment]
Looks good.

File: test/test_PokerPlayer4Functions.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from testing_base import *
+from source.PokerPlayer4Functions import *
+
+def test_4_Functions():
```
Comment (vondojr) [inline test/test_PokerPlayer4Functions.py:4]: Nice way to test all your functions in one!

**Review by andrewskerratt — APPROVED** [review]
wonderful John! You did a lot it all looks great

**Pull request merged**